### ISSUE TITLE
cannot import processing or turtle (possibly issue #3 redux)

### ISSUE THREAD

**Issue by sawula** [Calysto/skulpt_python#5]
- skulpt_python kernel seemed to install successfully through `pip install --upgrade skulpt_python` as revised at close of issue #3 
- tried to work through basic examples in Jupyter notebook using Skulpt Python kernel.  `import turtle` triggers the error  `ExternalError: TypeError: target is null on line 1` and `import processing` triggers the error message `ImportError: Failed to load remote module 'processing' on line 1`
- then went back and tried to install by cloning skulpt_python and running `python setup.py install`, but that didn't make any difference

Again, I'm on:
    Windows 8.1 Enterprise
    Notebook server 4.1.0
    Python 2.7.11
    Anaconda 2.5.0 (64-bit)
I believe I already have Metakernel, and Calysto installed via pip.

Thanks again!

**Comment by p5k12** [comment]
actually the error message for `import processing` is `ImportError: Failed to load remote module 'processing' on line 1`

**Comment by dsblank** [comment]
I'll have to dig into why the notebook on jupyter.cs.brynmawr.edu works... must be some differences there. Thanks for reporting!

**Comment by sawula** [comment]
Hi, Doug.

I'm up at Jupyter days in Boston.  Seems like a way around the differences
in dependencies/versions is for us to use Docker (www.docker.com).  btw
Kitematic.com is GUI for Docker.

There's also www.mybinder.org, but I believe Docker is more appropriate
because I want students to make lots of versions of your working notebook.
I'm not totally clear on this.

Could that be a plan for now?  I'm eager to use your Skulpt kernel.  It
sounds like creating a Docker container isn't that hard.

Thanks,

ms

**Comment by dsblank** [comment]
@p5k12 Cool... wish I were there!

Yes, docker (and mybinder) are very easy to set up. I'm tying to find the version I made that notebook with...

**Comment by sawula** [comment]
I wish you were here, too.  I kept scanning the room today thinking you would be!

**Comment by dsblank** [comment]
So, this seems to work for me:

``` python
from processing import *

def setup():
    size(400,400)
    noStroke()
    smooth()
    noLoop()

def draw():
    drawCircle(200,170,6)

def drawCircle(x, radius, level):
    tt = 128.0 * level / 4.0
    fill(tt)
    ellipse(x, 200, radius*2, radius*2)
    if level > 1:
        level = level - 1
        drawCircle(x - radius / 2, radius/2, level)
        drawCircle(x + radius / 2, radius/2, level)
run()
```

But I am getting an error importing turtle...

**Comment by dsblank** [comment]
What is your result of this:

``` shell
python3 -c "import skulpt_python; print(skulpt_python.__version__)"
```

**Comment by dsblank** [comment]
Oh, I see you are using Python2... maybe there is an issue there...

**Comment by dsblank** [comment]
Ah, yes... this is a Python2 issue. Looks like there may be code in the skulpt_python/kernel.py that is Python3 specific. Let me take a look...

**Comment by dsblank** [comment]
No, that wasn't it... I think it is not finding this file: http://localhost:8888/notebooks/processing/**init**.js

**Comment by dsblank** [comment]
I think I need to rebuild skulpt with the processing `__init__.js` in it...
